docs/modules/ROOT/pages/advanced-how-tos/installing/enabling-builds.adoc

```diff
@@ -15,4 +15,7 @@ Before users can create an application in {ProductName}, you must enable build p
 
 .Verification
 
-* To verify that you successfully enabled build pipelines, a user should try to xref:/how-tos/creating.adoc[create an application].
\ No newline at end of file
+* To verify that you successfully enabled build pipelines, a user should try to xref:/how-tos/creating.adoc[create an application].
+
+NOTE: The PipelineRun will run only for submitters who have permission to run PipelineRuns or who receive an `/ok-to-test` comment from an authorized user. +
+For further details, please refer to the https://pipelinesascode.com/docs/guide/running/[PipelinesAsCode documentation].
```

Comment:
I had the question "who is authorized" when I read this. While this is answered in the linked documentation, it isn't explicitly clear. Can we make that direction clearer? Maybe something like 
```suggestion
For further on PipelineRun permissions, please refer to the https://pipelinesascode.com/docs/guide/running/[PipelinesAsCode documentation].
```
?